content/docs/guides/ha_scale/perf-scale.md

```diff
@@ -0,0 +1,101 @@
+
+This test is performed on a Kubernetes cluster provisioned and managed by [Microsoft Azure Kubernetes Service](https://azure.microsoft.com/en-us/services/kubernetes-service/). The cluster has 100 Standard_DS2_v2 nodes (each node has 2 vCPUs and 7GB memory). It is ensured that there is only one OSM controller pod running with default resource quota during the test. OSM is configured in non-permissive traffic mode.
+
+## Mesh topology
+
+The testing mesh contains two types of service. One is a load generator (modified from [wrk2](https://github.com/giltene/wrk2)), which can send requests at specified rate, load balanced among all other services in the mesh. The load generator serves as the client side of a request. Throughout our test, there will be at most one load generator service. The other type of service is a simple echo application that replies with whatever it receives from an inbound request. It serves as the server side of a request. We can deploy multiple copies of the echo application in the mesh. In short, the mesh topology is a typical 1-to-N service architecture.
+
+<p align="center">
+  <img src="/docs/images/perf/mesh-topo.png" width="650"/>
+</p>
+
+Note that since both types of service are in the mesh and permissive mode is disabled, thus each request will be authenticated via mTLS. It means that requests go through the Envoy sidecar in both the client and the server.
+
+## Metrics
+
+The test result includes a lot of metrics collected. A clear definition of each metric can help avoid ambiguity. Below are the metric definitions we will use throughout this documentation:
+
+* Control plane
+    * Peak time CPU/Memory: the maximum CPU/memory usage during the testing process.
+    * Stable time CPU/Memory usage: the average CPU/memory usage when the control plane is off the peak time and becomes stable.
+* Data plane sidecar
+    * Peak time CPU/Memory: the maximum CPU/memory usage among all sidecar containers during the testing process.
+    * Stable time CPU/Memory usage: the average CPU/memory usage of all sidecar containers after they become stable.
+* Request latency: request roundtrip time. P50, P90, P99 and P999 of request latency are measured.
+
+The reason that peak time and stable time are measured individually is because of how the services are deployed. All the echo services will be deployed and added to the mesh at the same time. Therefore, the control plane receives a load spike after the deployment starts. All the Envoy sidecars are connecting to the control plane simutaneously. After that, the control plane becomes much more idle, since there is no more change to the mesh topology.
+
+## Performance
+
+In this test, we focus on the request overhead added by OSM as well as the computation resource consumptions. The RPS level of the load generator is calculated as the desired per pod RPS multiplies the number of server side pods. We test the per pod RPS at 500, 1000, 2000 and 4000 respectively.
```

Comment:
```suggestion
In this test, we focus on the latency overhead and resource consumption added by OSM's.  We currently test the per pod RPS at 500, 1000, 2000 and 4000 respectively.
```

Reply:
let's also add some smaller numbers. People may be interested in 10 and 100

Reply:
I will add 100. In our testing when the RPS is too low, there were too much noise and their results very close.